common/app/templates/headerInlineJS/membershipAccess.scala.js

```diff
@@ -14,8 +14,9 @@
 
         require([
             'common/utils/$',
-            'common/utils/ajax'
-        ], function($, ajax) {
+            'common/utils/ajax',
+            'common/modules/identity/api'
```

Comment:
This might look like I'm adding an extra JS library to preflight (`identity/api`), but my (possibly incorrect) understanding is that preflight already loads everything in common.js, which already includes [`identity/autosignin`](https://github.com/guardian/frontend/blob/e79f3da2/static/src/javascripts/bootstraps/common.js#L30) - and _that_ includes [`identity/api`](https://github.com/guardian/frontend/blob/e79f3da29/static/src/javascripts/projects/common/modules/identity/autosignin.js#L8) ...so I'm not making anything worse. I think. @davidrapson says so anyway.